theano/tensor/nnet/corr_gemm.c

```diff
@@ -203,6 +203,7 @@ PyArrayObject* corrMM(PyArrayObject* bottom,
         output = top;
         // valid correlation: im2col, then gemm
         // Iterate over batch
+        #pragma omp parallel for schedule(static)
```

Comment:
I think this is a good first version, but I have questions on details of omp.

If the blas version isn't multi-threads, then I'm pretty sure this is good. My question is if gemm use multi-threads, it would be bad if both the omp and the blas use the total number of threads in the computer.

Is you blas linked with multi-thread blas? How many threads do you see being used in top? Do it cap for a long time at the number of core or do it go higher?